roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -59,6 +59,9 @@
   register: oshawkular_create_cluster_role
   failed_when: "'already exists' not in oshawkular_create_cluster_role.stderr and oshawkular_create_cluster_role.rc != 0"
   changed_when: oshawkular_create_cluster_role.rc == 0
+  # AUDIT:changed_when_note: Checking the return code is insufficient
+  # here. We really need to verify the if the role even exists before
+  # we run this task.
```

Comment:
Any reason why the combination of changed_when and failed_when is not sufficient here? As long as the create isn't trying to create multiple objects, the combination of changed_when/failed_when should be sufficient here.

If creating multiple objects, we would need to be concerned about testing in this way, though.